<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReferenceWidget</class>
 <widget class="QWidget" name="ReferenceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_Publication">
     <property name="text">
      <string>The publication to cite:</string>
     </property>
     <property name="buddy">
      <cstring>publicationEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="PublicationEdit" name="publicationEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_CiteKey">
       <property name="text">
        <string>Citekey:</string>
       </property>
       <property name="buddy">
        <cstring>citeKeyEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="StringEdit" name="citeKeyEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Pages">
       <property name="text">
        <string>Pages:</string>
       </property>
       <property name="buddy">
        <cstring>pagesEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="StringEdit" name="pagesEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Chapter">
       <property name="text">
        <string>Chapter</string>
       </property>
       <property name="buddy">
        <cstring>chapterEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ChapterEdit" name="chapterEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>275</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StringEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/stringedit.h</header>
   <slots>
    <signal>widgetEnabled(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>PublicationEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/publicationedit.h</header>
  </customwidget>
  <customwidget>
   <class>ChapterEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/chapteredit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>showPublicationList()</slot>
  <slot>createReference()</slot>
  <slot>removeReference()</slot>
 </slots>
</ui>
